pkg/api/user_test.go

```diff
@@ -20,6 +22,55 @@ func TestUserApiEndpoint(t *testing.T) {
 			TotalCount: 2,
 		}
 
+		loggedInUserScenario("When calling GET on", "api/users/:id", func(sc *scenarioContext) {
```

Comment:
Sadly, `loggedInUserScenario` relies on Go Convey to build the scenario which didn't allow me to create this new test w/o it.